cmd/create/project.go

```diff
@@ -0,0 +1,94 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/lyft/flytestdlib/logger"
+	"gopkg.in/yaml.v2"
+
+	cmdCore "github.com/lyft/flytectl/cmd/core"
+)
+
+const (
+	projectShort = "Create project resources"
+	projectLong  = `
+Create the projects.(project,projects can be used interchangeably in these commands)
+::
+
+ bin/flytectl create project --id test --description test -p test
+Project Created
+
+::
+
+Create the project using yaml definition file
+
+::
+ bin/flytectl create project --file project.yaml 
```

Comment:
probably we should write the example yaml here too?
you can use
.. code-block:: yaml
   id: "project-unique-id"
   name: "Friendly name"
   description: "Some description for the project"

Reply:
why resolved?